apps/api/src/app/controllers/__tests__/__snapshots__/fetchManyViewModels.e2e.spec.ts.snap

```diff
@@ -87,6 +137,12 @@ Array [
       "type": "Point",
```

Comment:
This is an interesting modelling question. One option is to make placenames a separate resource and add edge connections. However, this would make creating the map more complex.

Currently, I didn't implement the 'properties' field from the GEOJSON spec on any of the SpatialFeature models. In their model, I believe this is where such things might go.

An interesting middle ground would be to generate the map via a one-step web of knowledge query and show the 'related resources' stacked on the map markers. Or maybe when you select a spatial feature on the map, you see related resources for that spatial feature. This could include Placenames.

However there are subtleties there that we should work through with Shane *et al.*.